src/pages/Lend/LoanSearchPage.vue

```diff
@@ -240,6 +241,7 @@ export default {
 				this.challengeData = goalsData?.goals?.values?.[0] || {};
 				const teamData = this.apollo.readQuery({ query: TeamInfoFromId, variables: { team_id: teamId } });
 				this.teamData = teamData?.community?.team || {};
+				setChallengeCookieData(this.cookieStore, teamId, this.teamData?.name);
```

Comment:
Following the general previous challenge behavior, shouldn't this be pushing the loan ID on add to basket?